index.js

```diff
@@ -177,13 +206,13 @@ handlers[Dota2.EGCBaseClientMsg.k_EMsgGCClientConnectionStatus] = function gcCli
 
 Dota2.Dota2Client = Dota2Client;
 
-require("./handlers/cache");
+//require("./handlers/cache");
```

Comment:
The commented-out handlers probably need to be enabled if you've tested that they work.